src/main/java/com/google/sps/servlets/CreateTaskServlet.java

```diff
@@ -21,19 +25,60 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String title = request.getParameter("title");
     String desc = request.getParameter("desc");
     String time = request.getParameter("time");
+    String username = request.getParameter("userN");
+    username = "bobby"; //assumption
+
+    String userID = "";
+    Entity loggedInUser = null;
 
+    // search for userID from userN in datastore 
     Datastore datastore = DatastoreOptions.getDefaultInstance().getService();
+    Query<Entity> query =
+        Query.newEntityQueryBuilder().setKind("User").build();
+    QueryResults<Entity> results = datastore.run(query);
```

Comment:
this runs in O(number of users), we could change this to O(1) by using userN as the datastore key then doing a datastore get